sidebars.js

```diff
@@ -6,22 +6,22 @@ module.exports = {
   sidebar: [
     'index',
     {
-      Overview: [
+      Architecture: [
         'overview/introduction',
         'overview/authorizer',
-        'overview/policy',
+        {
+          Policy: ['overview/policy', 'overview/policy-lifecycle'],
```

Comment:
I thought we wanted this flat

Reply:
I tried to mostly flatten and not have sections within sections. There were a few exceptions. If you want to flatten we could combine into a single longer page.

Reply:
Flattened / brought in some of the policy lifecycle stuff and linked to the rest